ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -161,9 +161,6 @@ impl Blockstore {
             .db
             .batch()
             .expect("Database Error: Failed to get write batch");
-        // delete range cf is not inclusive
```

Comment:
nit: Maybe mention range inclusiveness in doc comments, ie 
https://github.com/solana-labs/solana/blob/56d698de61e7b5b4b6045d748e17af2d42b8b8da/ledger/src/blockstore/blockstore_purge.rs#L149
to
```rust
/// over [`from_slot`, `to_slot`].
```